get-all-systems-with-domains.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+
+"""
+Script to provide a listing of all domains found within the system, paired with
+the org and space names they belong to.
+
+NOTE:  This script assumes you are logged into CF on the command line.
+"""
+
+import csv
+import json
+import subprocess
+
+from urllib import error, request
```

Comment:
this import isn't used

Reply:
Really need a `d'oh` emoji here in GitHub...  Same as the `csv` import remark above!